build.sbt

```diff
@@ -57,6 +59,7 @@ playBuildExtraPublish := {
 // Dependencies
 
 def playEbeanDeps = Seq(
+  "com.typesafe.play" %% "play-guice" % PlayVersion,
```

Comment:
Does the actual module code for play-ebean have guice-specific code? Is it possible to only use this in tests?